packages/ndla-ui/src/MessageBox/MessageBox.tsx

```diff
@@ -148,6 +181,15 @@ export const MessageBox = ({
         </CloseButtonWrapper>
       )}
     </Wrapper>
+
+    {links && (
+      //loops through the links passed in as properties if there are any and creates a working link for each of them
+      <LinkWrapper>
+        {links.map((x) => (
+          <Link href={x['href']}>{x['name']}</Link>
+        ))}
```

Comment:
Eg liker at navninga gir meining sjølv om det er ganske tydelig kva som skjer. Og om du typer dette riktig kan du bruke `link.href` og `link.name`.
```suggestion
        {links.map((link) => (
          <Link href={link['href']}>{link['name']}</Link>
        ))}
```